kwave/kspaceFirstOrder2D.py

```diff
@@ -295,13 +295,21 @@ def kspaceFirstOrder2D(
     # start the timer and store the start time
     TicToc.tic()
 
+    # Currently we only support binary execution, meaning all simulations must be saved to disk.
```

Comment:
An idea that is not a blocker for this PR: what if we add this to the error string or somewhere in the documentation? Explanation here deserves more than staying as a comment IMO.

Reply:
I agree. We should open an enhancement issue for the next point release.